src/fides/api/ctl/migrations/versions/5307999c0dac_remove_deprecated_data_uses_for_.py

```diff
@@ -0,0 +1,32 @@
+"""remove deprecated data uses for fideslang 1.4
+
+Revision ID: 5307999c0dac
+Revises: 76c02f99eec1
+Create Date: 2023-06-11 11:15:53.386526
+
+"""
+from alembic import op
+
+from loguru import logger
+from sqlalchemy import text
+from sqlalchemy.engine import Connection
+
+
+# revision identifiers, used by Alembic.
+revision = "5307999c0dac"
+down_revision = "76c02f99eec1"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    logger.info("Removing default data uses.")
+    bind: Connection = op.get_bind()
+    bind.execute(text("DELETE FROM ctl_data_uses WHERE is_default = TRUE;"))
+
+    # This leaves the data uses empty, but the step to seed them back comes _after_ these migrations
```

Comment:
The other option we have here is to add them back _right here_ in case this would case issues with other migrations down the line?

Although we would theoretically be aware of those problems at the time of adding that new migration, and be able to fix it there

Reply:
After thinking about this again, I don't think this is an issue logically. Considering the default taxonomy is _always_ seeded at the end of the migrations, this means that we should _never_ be writing migrations that rely on the default taxonomy tables having the default taxonomies in them at migration-time.

Given this fact, I don't think seeding them is necessary or even correct in this case

Reply:
I think that's fair; migrations should avoid ever depending on any imported modules 👍